lib/abiParser.ts

```diff
@@ -1,8 +1,10 @@
 import debug from "./debug";
 import { EvmType, VoidType, parseEvmType } from "./typeParser";
-import { yellow } from "chalk";
+import chalk from "chalk";
 import { MalformedAbiError } from "./errors";
 
+const { yellow } = chalk;
```

Comment:
please revert. Why do you prefer object destruction instead doing this on one go while importing?

Reply:
Unfortunately that format is no longer supported, see https://github.com/chalk/chalk/issues/215#issuecomment-338889272

Reply:
Uhh, that's horrifying but I guess we need to roll with it 👍

Reply:
Yeah, sorry about that. The maintainer of chalk seems quite adamant that it is the best way to do it :/